packages/bitmasks/bitmasks.1.1.0/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "david.allsopp@metastack.com"
+authors: [ "David Allsopp" ]
+license: "BSD-3-clause with OCaml linking exception"
+homepage: "https://metastack.github.io/bitmasks"
+dev-repo: "https://github.com/metastack/bitmasks"
+bug-reports: "https://github.com/metastack/bitmasks/issues"
+build: [ "jbuilder" "build" "@install" ]
+build-doc: [
+  ["jbuilder" "build" "@doc"]
+  ["cp" "-r" "_build/default/_doc/bitmasks" "_build/default/_doc/index.html" "_build/default/_doc/odoc.css" "%{doc}%/"]
+]
+build-test: ["jbuilder" "runtest"]
+depends: [
+  "jbuilder" {build}
+  "odoc" {build & doc}
```

Comment:
I'm not sure of the semantics of ```build``` and ```test``` or ```doc``` together. I remember having something weird when the two were mixed. Does it not install ```odoc``` when there is no ```--build-doc``` present ? Just to be sure.

Reply:
I'll test that in a bit

Reply:
There doesn't seem to be anything, but it's also not necessary